lib/activerecord-multi-tenant/multi_tenant.rb

```diff
@@ -1,18 +1,12 @@
 require 'request_store'
 
 module MultiTenant
-  @@tenant_klass = nil
```

Comment:
Class variables here did not seem to be necessary, and goes against the philosophy of defining the tenant class per module